_includes/pbc/all/install-features/202505.0/install-the-spryker-core-feature.md

```diff
@@ -0,0 +1,1527 @@
+
+Make sure you don't use the same Redis database for Yves and Zed sessions.
+
+{% endinfo_block %}
+
+**config/Share/config_default.php**
+
+```php
+<?php
+
+use Spryker\Shared\SessionRedis\SessionRedisConstants;
+
+$config[SessionRedisConstants::YVES_SESSION_REDIS_SCHEME] = 'tcp';
+$config[SessionRedisConstants::YVES_SESSION_REDIS_HOST] = '127.0.0.1';
+$config[SessionRedisConstants::YVES_SESSION_REDIS_PORT] = 6379;
+$config[SessionRedisConstants::YVES_SESSION_REDIS_PASSWORD] = false;
+$config[SessionRedisConstants::YVES_SESSION_REDIS_DATABASE] = 1;
+```
+
+
+4. If the file system is used as a session storage, add the following configuration:
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Spryker\Shared\Session\SessionConfig;
+use Spryker\Shared\Session\SessionConstants;
+use Spryker\Shared\SessionFile\SessionFileConfig;
+use Spryker\Shared\SessionFile\SessionFileConstants;
+
+// ---------- Session
+$config[SessionConstants::YVES_SESSION_SAVE_HANDLER] = SessionFileConfig::SESSION_HANDLER_FILE;
+$config[SessionFileConstants::YVES_SESSION_TIME_TO_LIVE] = SessionConfig::SESSION_LIFETIME_1_HOUR;
+$config[SessionFileConstants::YVES_SESSION_FILE_PATH] = session_save_path();
+```
+
+#### Set up SecurityBlocker
+
+Pass the locale used in the login check path to `SecurityBlocker`:
+
+**src/Pyz/Yves/SecurityBlockerPage/SecurityBlockerPageConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\SecurityBlockerPage;
+
+use SprykerShop\Yves\SecurityBlockerPage\SecurityBlockerPageConfig as SprykerSecurityBlockerPageConfig;
+
+class SecurityBlockerPageConfig extends SprykerSecurityBlockerPageConfig
+{
+    /**
+     * @var bool
+     */
+    protected const USE_EMAIL_CONTEXT_FOR_LOGIN_SECURITY_BLOCKER = false;
+
+    /**
+     * @return bool
+     */
+    public function isLocaleInCustomerLoginCheckPath(): bool
+    {
+        return true;
+    }
+
+    /**
+     * @return bool
+     */
+    public function isLocaleInAgentLoginCheckPath(): bool
+    {
+        return true;
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that, when the login form for the customer or agent is submitted, the URL it uses contains a locale code. Examples of default values:
+* For customer: `/de/login_check`
+* For agent: `/de/agent/login_check`
+
+{% endinfo_block %}
+
+{% info_block infoBox "" %}
+
+All locale-related configs in `CustomerPage`, `AgentPage`, and `SecurityBlockerPage` are deprecated; in future releases, only locale-specific URLs will be used.
+
+{% endinfo_block %}
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Spryker\Shared\Log\LogConstants;
+use Spryker\Yves\Log\Plugin\Log\YvesSecurityAuditLoggerConfigPlugin;
+
+$config[LogConstants::LOG_FILE_PATH_YVES] = 'php://stdout';
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_YVES] = [
+    YvesSecurityAuditLoggerConfigPlugin::class,
+];
+```
+
+{% info_block warningBox "Verification" %}
+
+After finishing the installation, make sure the following applies:
```

Comment:
Does it make sense to move this verification to where it can be executed? Here, you can mention that the verification will be done at a later step

Reply:
To be honest, I don't know this topic, looks like it's a proper place.